enhancements/network/egress-router.md

```diff
@@ -0,0 +1,100 @@
+---
+title: OVN Egress Router Support
+authors:
+  - "@dmellado"
+  - "@jdesousa"
+reviewers:
+  - "@squeed"
+  - "@danwinship"
+approvers:
+  - "@knobunc"
+creation-date: 2020-08-26
+last-updated: 2020-08-26
+status: provisional
+---
+
+# OVN Egress Router Support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Egress traffic is traffic going from OpenShift pods to external systems,
+outside of OpenShift. There are two main options for enabling egress traffic:
+Allow access to external systems from OpenShift physical node IPs or use egress
+router.
+
+In enterprise environments, egress routers are often preferred. They
+allow granular access from a specific pod, group of pods, or project to an
+external system or service. Access via node IP means all pods running on a
+given node can access external systems.
+
+An egress router is a pod that has two interfaces (eth0) and (macvlan0). Eth0
+is on the cluster network in OpenShift (internal) and macvlan0 has an IP and
+gateway from the external physical network.
+
+Pods can access the egress router service thus enabling them to access
+external services. The egress router acts as a bridge between pods and an
+external system.
+
+Traffic going out the egress router goes via node, but it will have the MAC
+address of the macvlan0 interface inside the egress router.
+
+Nowadays this is not supported when using ovn-kubernetes, mainly due to the
+lack of macvlan support, and its implementation is needed to ensure
+feature-parity with openshift-sdn.
+
+## Motivation
+
+* The egress router images allow people to run containers that have defined
+  external addresses.
+
+  Using network policies you can control the scope of who can talk to egress
+  routers and thus, when they send traffic outside the cluster it's possible
+  to identify the containers in the cluster which are sending it. This allows
+  to use external firewalls forauditing and filtering.
+
+  Openshift-sdn is able to request a macvlan interface with a magic annotation.
+  As this is not possible using ovn-kubernetes, this would be done using multus
```

Comment:
Again, it's not that it's "not possible", it's that we don't _want_ to do it that way any more.